databases/week3/Week3-homework/additional_queries.sql

```diff
@@ -0,0 +1,98 @@
+-- Get meals that has a price smaller than a specific price fx 90
+SELECT 
+    *
+FROM
+    meal
+WHERE
+    price < 350;
+
+-- Get meals that still has available reservations
+SELECT 
+    meal.id,
+    meal.title,
+    meal.max_reservations,
+    reservation.id AS reservation_id,
+    reservation.number_of_guests
+FROM
+    meal
+        LEFT JOIN
+    reservation ON reservation.meal_id = meal.id
+WHERE
+    meal.max_reservations > (SELECT 
+            SUM(number_of_guests)
+        FROM
+            reservation
+        WHERE
+            reservation.meal_id = meal.id)
+ORDER BY meal.id;
+
+-- Get meals that partially match a title. Rød grød med will match the meal with the title Rød grød med fløde
+SELECT 
+    *
+FROM
+    meal
+WHERE
+    title LIKE '%burger%';
+
+-- Get meals that has been created between two dates
+SELECT
+  *
+FROM
+  meal
+WHERE
+  created_date > '2021-07-12'
+  AND created_date < '2021-08-15';
+
+-- Get only specific number of meals fx return only 5 meals
+SELECT
+  *
+FROM
+  meal
+ORDER BY
+  price
+LIMIT
+  5;
+
+-- Get the meals that have good reviews
+SELECT
+  *
+FROM
+  meal
+WHERE
+  id IN (
+    SELECT
+      meal_id
+    FROM
+      review
+    WHERE
+      stars = 4
+  );
```

Comment:
Here what is there range for good stars, bad & average stars? let's say good= 4 , how about bad & average? 1 ,2,3 or what if the stars are 5 or more? So better to set a range to specify the star category. Here you could say `WHERE stars>4` , meaning all given stars above 4 are categorized as good.